flake.nix

```diff
@@ -17,6 +17,37 @@
   inputs.nixdoc-to-github.flake = false;
   inputs.nixdoc-to-github.url = "github:fricklerhandwerk/nixdoc-to-github";
 
+  inputs.hillingar.inputs.nixpkgs.follows = "nixpkgs";
+  inputs.hillingar.inputs.flake-utils.follows = "flake-utils";
+  inputs.hillingar.inputs.opam-repository.follows = "opam-repository";
+  inputs.hillingar.inputs.opam-overlays.follows = "opam-overlays";
+  inputs.hillingar.inputs.mirage-opam-overlays.follows = "mirage-opam-overlays";
+  inputs.hillingar.inputs.opam-nix.follows = "opam-nix";
+  inputs.hillingar.inputs.opam2json.follows = "opam-nix/opam2json";
+  # TODO(linj) handle my patches proerly: upstream them
+  inputs.hillingar.url = "github:linj-fork/hillingar?ref=local/debug-clean";
+  inputs.opam-nix.inputs.nixpkgs.follows = "nixpkgs";
+  inputs.opam-nix.inputs.flake-utils.follows = "flake-utils";
+  inputs.opam-nix.inputs.opam2json.inputs.systems.follows = "systems";
+  inputs.opam-nix.inputs.opam-repository.follows = "opam-repository";
+  inputs.opam-nix.inputs.opam-overlays.follows = "opam-overlays";
+  inputs.opam-nix.inputs.mirage-opam-overlays.follows = "mirage-opam-overlays";
+  # update opam-nix to fix eval error of new Nixpkgs: attribute 'overrideScope'' missing
+  inputs.opam-nix.url = "github:tweag/opam-nix";
```

Comment:
This one is using IFD which will prevent us to merge this PR. Is there any workaround (for example including some generated file in git)?